src/servant/play_sfx.h

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+void ProcessSfxState(Entity* entity) {
+    switch (entity->step) {
+    case 0:
+        entity->flags = FLAG_UNK_20000 | FLAG_KEEP_ALIVE_OFFCAMERA;
+        if (D_8003C704 != 0) {
+            g_PlaySfxStep = 99;
+            DestroyEntity(entity);
+            return;
+        }
+        if (g_api.CdSoundCommandQueueEmpty() != 0) {
+            g_api.PlaySfx(SET_UNK_10);
+            entity->step++;
+        }
+        break;
+
+    case 1:
+        if (g_api.func_80133950() != 0) {
+            entity->step++;
+        }
+        break;
+    case 2:
+        g_api.PlaySfx(entity->params);
+        entity->step++;
+        break;
+
+    case 3:
+        if (g_api.func_80131F68() != 0) {
+            entity->step++;
+        }
+        break;
+
+    case 4:
+        if (g_api.func_80131F68() == 0) {
+            entity->step++;
+        }
+        break;
+
+    case 5:
+        if ((*(s32*)&D_8003C708.flags & 0x60) != 0) {
```

Comment:
I know this predates your PR but would you mind putting the psp version in a scratch? I suspect it's an optimization and it's actually something like 
```
if(D_8003C708.flags && (D_8003C708.unk2 & 0x60))
```
but that's not it

Reply:
This function is ifndef'd out of the PSP build so I don't think it's part of that release?

Reply:
Here's the PSX scratch if that helps
https://decomp.me/scratch/yEfKI